### PR TITLE
DO NOT MERGE: david patches for the pr #1918

### PR DESCRIPTION
**Pull request by david-caro**

### CODE REVIEW

**Comment by david-caro** [comment]
Still failing some tests, but fixed a bunch of them so far

**Comment by david-caro** [comment]
@jacquerie Keep this in mind ;)

**Comment by david-caro** [comment]
This actually passed all the acceptance tests locally \o/

**Comment by david-caro** [comment]
Noting that the acceptance tests do not really check that the records are actually ingested at the end, so the last 'verification' errors due to the schema do not  fail the tests... (I'll try to add that tomorrow)

**Comment by david-caro** [comment]
Getting a bunch of unit tests errors though :/, anyhow... too late today...

File: inspirehep/bat/pages/holding_panel_author_list.py [hunk 1]
```diff
@@ -29,37 +29,53 @@
...
+    WebDriverWait(Arsenic(), 10).until(
+        EC.visibility_of_element_located(
+            (By.XPATH, '//a[@class="title ng-binding ng-scope"]')
+        )
+    ).click()
+    WebDriverWait(Arsenic(), 10).until(
+        EC.visibility_of_element_located(
+            (By.XPATH, '(//div[@class="detail-panel"])[1]')
+        )
+    )
```
Comment (jacquerie) [inline inspirehep/bat/pages/holding_panel_author_list.py]: I was waiting for YAPF to fix these kind of things all in one go, but ok.

Comment (david-caro) [inline inspirehep/bat/pages/holding_panel_author_list.py]: it annoys me that my editor is complaining all the time...

**Pull request closed without merging**